### PR TITLE
Proper condition-free CI

### PR DESCRIPTION
**Pull request by szepeviktor**
Untangle Travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,13 +15,11 @@ cache:
...
-  - 7.1
-  - 7.2
-  - 7.3
-  - 7.4
-  - "nightly"
+  - "7.0"
+  - "7.1"
+  - "7.2"
+  - "7.3"
+  - "7.4"
```
Comment (szepeviktor) [inline .travis.yml:22]: These are strings not floating point numbers.

File: .travis.yml [hunk 2]
```diff
@@ -54,25 +69,12 @@ before_install:
...
   # Set up environment using Composer.
-  - |
-    if [[ $TRAVIS_PHP_VERSION == "nightly" ]]; then
-      # Not all PHPUnit dependencies have stable releases yet allowing for PHP 8.0.
-      travis_retry composer install --no-interaction --ignore-platform-reqs
-    else
-      # Do a normal dev install in all other cases.
-      travis_retry composer install --no-interaction
-    fi
+  - travis_retry composer update --no-interaction
```
Comment (szepeviktor) [inline .travis.yml:72]: We have no click file!

**Comment by szepeviktor** [comment]
@jrfnl 🍏

**Comment by jrfnl** [comment]
@szepeviktor Hiya, Thanks for the PR, but it is unclear to me what you are trying to fix.

**Comment by szepeviktor** [comment]
> what you are trying to fix.

Removing conditions.
I do not know how to justify it harder.

**Comment by szepeviktor** [comment]
...conditions directly cause branching which is error prone.

**Comment by szepeviktor** [comment]
If you do not see the bloody 👹 teeth of conditions simply close this one.

**Comment by szepeviktor** [comment]
```php
if () {
	if () {
		if () {
			if () {
			}
		}
	}
}
```

🦈 attack!

**Comment by jrfnl** [comment]
@szepeviktor I agree that complex scripts with lots of condition aren't great, but in this case, there are only two simple and well documented conditions and by removing those and moving the alternative run commands elsewhere, the cognitive complexity of the script is raised as the logic of what's being done is now split up to different locations. Either way, that's just my opinion.

**Comment by szepeviktor** [comment]
All right.

**Pull request closed without merging**